hydroshare/settings.py

```diff
@@ -189,8 +189,8 @@ def _(s): return s # noqa
 # OIDC_OP_USER_ENDPOINT = "https://auth.cuahsi.io/realms/CUAHSI/protocol/openid-connect/userinfo"
 # OIDC_RP_SIGN_ALGO = "RS256"
 # OIDC_OP_JWKS_ENDPOINT = "https://auth.cuahsi.io/realms/CUAHSI/protocol/openid-connect/certs"
-# OIDC_RP_CLIENT_ID = 'hydroshare'
-# OIDC_RP_CLIENT_SECRET = 'Ya4GzskPjEmvkX6cL8w3X0sQPNW6CwkM'
+OIDC_RP_CLIENT_ID = 'hydroshare'
```

Comment:
These keys need to be present is settings.py because the OIDCAuthentication implementation expects the settings to be declared (needed for running unit tests).